packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -45,29 +46,32 @@ import {
 	checkoutWithContent,
 	flexTreeViewWithContent,
 	insert,
+	jsonSequenceRootSchema,
 	toJsonableTree,
 } from "../utils.js";
 
-// number of nodes in test for wide trees
+// Number of nodes in test for wide trees.
 const nodesCountWide = [
 	[1, BenchmarkType.Measurement],
 	[100, BenchmarkType.Perspective],
-	[500, BenchmarkType.Measurement],
+	[1000, BenchmarkType.Measurement],
+	[10000, BenchmarkType.Measurement],
 ];
-// number of nodes in test for deep trees
+// Number of nodes in test for deep trees.
+// TODO: We currently run into a "maximum call stack size exceeded" error if we increase the node count,
+// due to recursive calls when calling flexTreeViewWithContent, or calling assert.deepEqual.
+// To include tests with higher node counts, we will likely need to replace these parts of the code to be done iteratively.
```

Comment:
This is not so much an issue with these tests, but a major limitation of a large portion of our tree processing code. The encoders, and decoders, editing code and more will all fail with really deep tress.

Reply:
Updated comment mentioning the current limitations of a large portion of our tree processing code.